woocommerce/woocommerce-ios
# Blaze: Display new bottom sheet upon campaign creation success

Closes: #11736 


## Description

This PR updates the navigation when campaign creation succeeds. Previously, we displayed the campaign list upon success and presented a celebration bottom sheet if it's the first campaign created on the app. Following the confirmation from the designer [p1705640966543879-slack-C03L1NF1EA3] - we will discard this flow and present the new success bottom sheet instead.

For simplicity, this new navigation is updated for both the web and native flows. 

## Testing instructions

- Log in to a store that's eligible for Blaze.
- Select Promote button in the Blaze section on the Dashboard screen.
- On the campaign creation form, tap Confirm Details.
- On the payment screen, tap Submit Campaign.
- Confirm the following:
  - A loading screen is displayed briefly, then the creation flow is dismissed.
  - A new bottom sheet is displayed for the success. Confirm that the UI matches the design.
  - The campaign list is no longer displayed afterward.

Repeat the above steps for the entry points from the product form and the campaign list to ensure that the navigation works properly for all cases.

Feel free to disable the feature flag `blazei3NativeCampaignCreation` and repeat the tests - the new success flow should be applied now.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/5533851/bb9acd14-6594-4ac0-8c5f-8ffa43eaae71



---
